tests/Queries/QueryBuilderFactoryTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Mado\QueryBundle\Tests\Objects;
+
+use Mado\QueryBundle\Queries\QueryBuilderFactory;
+use Mado\QueryBundle\Vocabulary\Operators;
+use PHPUnit\Framework\TestCase;
+
+class QueryBuilderFactoryTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->manager = \Doctrine\ORM\EntityManager::create(array(
+            'driver' => 'pdo_sqlite',
+            'path' => __DIR__ . '/data/db.sqlite',
+        ),
+        \Doctrine\ORM\Tools\Setup::createAnnotationMetadataConfiguration(
+            array(__DIR__."/src"),
+            true
+        ));
+    }
+
+    public function testCanFilterThanksToOperators()
```

Comment:
testCanFilterByOperators or testCanFilterWithOperators

Reply:
ByOperator is better

Reply:
Ok ByOperator